sdk/src/main/java/com/kin/ecosystem/ViewExt.kt

```diff
@@ -28,4 +28,17 @@ inline fun <T : ValueAnimator> T.withEndAction(crossinline action: T.() -> Unit)
             action()
         }
     })
+}
+
+inline fun <T : ValueAnimator> T.withActions(crossinline startAction: T.() -> Unit, crossinline endAction: T.() -> Unit) {
```

Comment:
i'm not 100% sure, but I think you can return `T` instead of `Unit`, and then you can act on a specific object without specifying it (like `when` operator)
so you're code in
`withActions(startAction = {
                    earnOrderRecyclerView.visibility = VISIBLE
                }, endAction = {
                    spendOrderRecyclerView.visibility = GONE
                })
`
can become:
`
spendOrderRecyclerView.withActions(startAction = {
                   visibility = VISIBLE
                }, endAction = {
                   visibility = GONE
                })
`